themes/digital.gov/layouts/shortcodes/do-dont-table.html

```diff
@@ -0,0 +1,42 @@
+{{/* 
+
+{{< do-dont-table caption="This is a table" >}}
+  {{< do-dont-row do="do this today" dont="don't do this today" >}}
+  {{< do-dont-row do="open the door for those who need a hand" dont="use inappropriate language in the chats" >}}
+  {{< do-dont-row do="read the directions before using" dont="forget to wash your hands for 20 seconds" >}}
+{{< /do-dont-table >}}
+
+When setting up a **do-dont-table**, you need to make sure you have the correct formatting.
+
+* Start with a a **do-dont-table** then nest **do-dont-row**
+* It is best practice to provide a caption which describes the table contents, should be provided.
+* When you don't provide a matching pair for **do** or **don't**, the table row will not appear. Make sure you have both.
+* Make sure to close the **do-dont table** with a closing tag, it is not needed for the **do-dont-row**
+
+*/}}
+
+
+<table class="usa-table usa-table--borderless dg-do-dont-table usa-table--stacked">
+  <caption>{{- if (.Get "caption") -}}{{ .Get "caption" }}{{- else -}} {{- end -}}</caption>
```

Comment:
Wrapping the `<caption>` element will prevent it from being rendered empty if caption is left out. I'd also remove the `{{- else -}}` unless it's necessary.

```suggestion
  {{- if (.Get "caption") -}}
    <caption>{{ .Get "caption" }}</caption>
  {{- end -}}
```

Reply:
Got it, I got this opposite. I figured it would be best to have it empty if present.